bin/verilator

```diff
@@ -1144,13 +1144,12 @@ value of 0.
 
 =item --output-split-cfuncs I<statements>
 
-Enables splitting functions in the output .cpp files into multiple
-functions.  When a generated function exceeds the specified number of
-operations, a new function will be created.  With --output-split, this will
-enable GCC to compile faster, at a small loss in performance that gets
-worse with decreasing split values.  Note that this option is stronger than
---output-split in the sense that --output-split will not split inside a
-function.
+Enables splitting functions in the output .cpp files into multiple functions.
+When a generated function exceeds the specified number of operations, a new
+function will be created.  With --output-split, this will enable the C++
+compiler to compile faster, at a small loss in performance that gets worse with
```

Comment:
Looks wide, please fill paragraph.

Reply:
This is 79 wide which is less than 80?

Reply:
Ah, my fill-column is 75, I'll add that setting to bin/verilator for others unless you think we should up it.

Reply:
I don't mind either value, making it explicit with a note in some internals doc would help though [I'm using vim so the emacs inlines don't help me much :)].